package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "colorator",
+  "version": "0.0.1",
+  "description": "Convert one terminal color scheme to another",
+  "main": "index.html",
+  "devDependencies": {
+    "typescript": "3.2.2"
+  },
+  "scripts": {
+    "tsc": "tsc -p .",
+    "test": "npm run tsc"
```

Comment:
In `test` script stanza in package.json I write a start of tsc (typescript compiler).